mobius-core/src/commonMain/kotlin/Next.kt

```diff
@@ -83,23 +83,41 @@ class Next<M, F> internal constructor(
     }
 
     companion object {
-        /** Create a Next that updates the model and dispatches the optional set of effects. */
+
+        /** Create a Next that updates the model and dispatches the optional effects. */
         @JvmStatic
         @JvmOverloads
         @JsName("next")
-        fun <M, F> next(model: M, effects: Set<F> = emptySet()): Next<M, F> {
-            return Next(model, effects.toSet())
+        fun <M, F> next(model: M, effects: Set<F>): Next<M, F> {
+            return next(model, effects)
         }
 
-        /** Create a Next that doesn't update the model but dispatches the supplied effects. */
+        /** Create a Next that updates the model and dispatches the optional effects. */
         @JvmStatic
+        @JvmOverloads
+        fun <M, F> next(model: M, vararg effects: F): Next<M, F> {
+            return next(model, effects.toSet())
+        }
+
+        /** Create a Next that doesn't update the model but dispatches the supplied effect. */
+        @JvmStatic
+        @JvmOverloads
         @JsName("dispatch")
         fun <M, F> dispatch(effects: Set<F>): Next<M, F> {
-            return Next(null, effects.toSet())
+            return dispatch(effects)
+        }
+
+        /** Create a Next that doesn't update the model but dispatches the supplied effects. */
+        @JvmStatic
+        @JvmOverloads
+        fun <M, F> dispatch(vararg effects: F): Next<M, F> {
+            return dispatch(effects.toSet())
         }
 
         /** Create an empty Next that doesn't update the model or dispatch effects. */
         @JvmStatic
+        @JvmOverloads
+        @JsName("noChange")
```

Comment:
```suggestion
```

As this function does not take any arguments, these annotations are useless here.